class ErrorAplicacion(Exception):
    def __init__(self):
        super().__init__("Manejo de errores de la Aplicaci√≥n:")